app/code/core/Mage/Core/Model/Resource/File/Storage/File.php

```diff
@@ -47,6 +47,9 @@ class Mage_Core_Model_Resource_File_Storage_File
      */
     protected $_ignoredFiles;
 
+    /** @var resource */
+    protected $filePointer;
```

Comment:
@Flyingmana we are not following the Magento convention that private/protected files have underscore prefix?